demo/api/src/news/blocks/news-detail.block.ts

```diff
@@ -24,7 +25,7 @@ class NewsDetailBlockData extends BlockData {
 class NewsDetailBlockInput extends BlockInput {
     @BlockField({ nullable: true })
     @IsUUID()
-    @IsOptional()
+    @IsUndefinable()
     id?: string;
```

Comment:
idn't nullable here correct?

(we don't support partial update for block input)

Reply:
Block fields are usually `undefined`, not `null`. Therefore I chose `@IsUndefinable()`. I've also tested it with an empty input, and it works.

> (we don't support partial update for block input)

Yes, here setting it to undefined makes it undefined, which is correct since it's "just" a JSON field.